.azure/applications/update-subject-resources-job/prod.bicepparam

```diff
@@ -0,0 +1,10 @@
+using './main.bicep'
+
+param environment = 'prod'
+param location = 'norwayeast'
+param imageTag = readEnvironmentVariable('IMAGE_TAG')
+param jobSchedule = '0 * * * *' // Runs every hour
```

Comment:
How often should we run this job @elsand ? 🤔

Reply:
Let's start with every 5 minutes